frontend/packages/dev-console/src/components/import/image-search/ImageStream.tsx

```diff
@@ -0,0 +1,192 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Alert } from '@patternfly/react-core';
+import { useField, useFormikContext, FormikValues } from 'formik';
+import { k8sGet, K8sResourceKind } from '@console/internal/module/k8s';
+import { ImageStreamTagModel } from '@console/internal/models';
+import { DropdownField } from '../../formik-fields';
+import { ImageStreamProps } from '../import-types';
+import {
+  getSuggestedName,
+  getPorts,
+  makePortName,
+  normalizeBuilderImages,
+  NormalizedBuilderImages,
+  BuilderImage,
+  registryType,
+  builderImagesNamespace,
+} from '../../../utils/imagestream-utils';
+import './ImageStream.scss';
+
+const ImageStream: React.FC<ImageStreamProps> = ({ imageStreams }) => {
+  const projectList = {};
+
+  imageStreams.reduce((acc, { metadata: { namespace } }) => {
+    if (!acc[namespace]) {
+      acc[namespace] = namespace;
+    }
+    return acc;
+  }, projectList);
+
+  const [imageNamespace] = useField('imageStream.namespace');
```

Comment:
You do not need to use `useField` here if you're just going to use `value` property of it. `useField` is helpful if you're trying to create a custom field and you need to use things like `field.value`, `field.onChange` and `field.onBlur`.

Reply:
Its better to use `values`